src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -464,7 +498,7 @@ public int getInt(int index) throws SQLServerException {
     public int getInt(String sCol) throws SQLServerException {
         loggerExternal.entering(getClassNameLogging(), "getInt", sCol);
         checkClosed();
-        Integer value = (Integer) getValue(findColumn(sCol), JDBCType.INTEGER);
+        Integer value = (Integer) getValue(findColumn(sCol, false), JDBCType.INTEGER);
```

Comment:
It seems nit picky, but could you either: 1) define two boolean constants, IS_SETTER and IS_GETTER, or 2) use an Enum, for the parameter passed to findColumn()?

Reading this diff it was like a murder mystery until the end of the file; what is that parameter going to mean?

Reply:
@brettwooldridge Thank you for reviewing this! I will push in the next commit with the change to something more defining like you suggested.